CHANGELOG.rst

```diff
@@ -1,5 +1,58 @@
 Changelog
 =========
+1.9.1 (2020-07-06)
+------------------
+MLflow 1.9.1 includes several major features and improvements:
+
+Features:
+
+- Search Registered Models API (#2951, @mparkhe)
```

Comment:
We may not want to include this just yet (a bunch of fixes/implementation details for this didn't make 1.9.1)